dev/TreeView/TreeViewItem.xaml

```diff
@@ -16,14 +16,18 @@
         <Setter Property="BorderThickness" Value="{ThemeResource TreeViewItemBorderThemeThickness}"/>
         <Setter Property="GlyphBrush" Value="{ThemeResource TreeViewItemForeground}"/>
         <Setter Property="MinHeight" Value="{ThemeResource TreeViewItemMinHeight}"/>
+        <contract7Present:Setter Property="CornerRadius" Value="{ThemeResource OverlayCornerRadius}" />
         <Setter Property="Template">
             <Setter.Value>
                 <ControlTemplate TargetType="local:TreeViewItem">
-                    <Grid x:Name="ContentPresenterGrid" Margin="0,0,0,0" 
+                    <Grid x:Name="ContentPresenterGrid"
                         Background="{TemplateBinding Background}" 
                         BorderBrush="{TemplateBinding BorderBrush}"
                         BorderThickness="{TemplateBinding BorderThickness}"
-                        contract7Present:CornerRadius="{TemplateBinding CornerRadius}">
+                        Height="{TemplateBinding MinHeight}"
```

Comment:
>Height [](http://example.com/codeflow?start=24&length=6)

Will this force the item to be minheight and not let us size to content ?

Reply:
The `MinHeight` setter property is currently set to `TreeViewItemMinHeight` so one should be able to size the content by overwriting that resource.

Reply:
Oh, realized in the `TreeViewItemTemplateSelectorTest` that it is unreasonable to expect developers to set `MinHeight` to resize content. I've tweaked the template using padding on `ContentPresenterGrid` to get the correct item height.